### DISCUSSION TITLE
ピンソケットにはんだが入り込んでしまいました。

### DISCUSSION THREAD

**Discussion by sumomo-momo** [Yowkees/keyball#441] [Q&A]
keyball61を購入させていただきました。
promicro用のピンソケットの高さや品番、購入方法などを教えていただきたいです。keyball39と同じ品番でしょうか？
新しい部品を買わなくてもよい解決方法があればそれを教えていただきたいです。

いったん最後まで組み立てた後に、トラックボールとVと３のキーが反応せず、はんだが甘かったかなと思ってやり直していたらピンソケットにはんだが入ってしまいました。３か所ほど穴の中にはんだが入ってしまったようで、promicroに取り付けたピンヘッダがしっかり刺さりません。ご対応よろしくお願いいたします。

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

Keyballシリーズで使用しているピンソケットは同じ品番のものを使用しています。

Keyballシリーズを購入していただいた方限定にピンソケットの商品ページがございますので、白銀ラボのウェブショップにありますお問い合わせフォームから再度ご連絡していただいてもよろしいでしょうか。
